Clarification allowedDeviation for corner cutting (#415)
VDA5050/VDA5050: VDA5050_EN.md

### 6.12.2 Traversal of nodes and entering/leaving edges, triggering of actions

The AGV decides on its own, when a node should count as traversed.

> As a decision support, the AGV's control point should be within the node's `allowedDeviationXY` and its orientation within `allowedDeviationTheta`.

Reviewer: ```suggestion
A requirement for the traversal is that the mobile robot's control point should be within the node's `allowedDeviationXY` and its orientation within `allowedDeviationTheta`.
```
